src/vendor/wordpress/styles/_jetpack-blocks.scss

```diff
@@ -154,19 +154,37 @@
     clear: left; // 1
     font-weight: font-weight.$medium; // 2
     margin-inline-end: 1ch; // 3
+
+    /// If there is no field value, don't bother displaying the field name.
+    &:has(+ .field-value:empty) {
+      display: none;
+    }
   }
 
-  /// 1. Field values appear slightly muted.
-  /// 2. Because values may break onto new lines, we need to use bottom margin
-  ///    to continue vertical rhythm. We only apply this when the field value is
-  ///    the last of its type: We can't use `:last-child` because it is often
-  ///    followed by an empty paragraph. (This can be simplified once
-  ///    `margin-trim` is widely supported!)
+  /// Field values appear slightly muted.
+
   .field-value {
-    color: var(--theme-color-text-muted); // 1
+    color: var(--theme-color-text-muted);
+
+    /// Empty field values need to clear floats to maintain margin.
+    &:empty {
+      clear: left;
+      float: none;
+
+      /// If the browser supports `:has()`, we can hide the field value entirely
+      /// since the field name will be hidden as well.
+      @supports selector(:has(+ :empty)) {
+        display: none;
+      }
```

Comment:
I'm a little confused about why we don't always hide this? I'm assuming we want to keep this in sync with its label, but it seems like if it's empty we could just always hide it.

I'm guessing there's some layout reason related to how our floats work, so feel free to ignore. I just figured I'd mention it in case it sparked an idea.

I also kind of wonder if doing something like this would be good as a fallback?

```css
@support not selector(:has(+ empty)) {
  &::before {
    content: 'No Value'
  }
}
```

I'm not sure... just more food for thought!

Reply:
Oh, I'm guessing we still need to display the element so that its bottom margin applies? That makes sense.

Reply:
Exactly, we need to keep its bottom margin around.

I thought about the pseudo element idea, but in the past I've received feedback that CSS `content` doesn't work with automated translation tools. I figured I'd keep this presentational.